src/commands/node.mjs

```diff
@@ -2316,8 +2362,1062 @@ export class NodeCommand extends BaseCommand {
+          await sleep(60000)
+          const accountMap = getNodeAccountMap(config.allNodeIds)
+          // update map with current account ids
+          accountMap.set(config.nodeId, config.newAccountNumber)
+
+          // update _nodeClient with the new service map since one of the account ids has changed
+          await this.accountManager.refreshNodeClient(config.namespace)
+
+          // send some write transactions to invoke the handler that will trigger the stake weight recalculate
+          for (const nodeId of config.allNodeIds) {
+            const accountId = accountMap.get(nodeId)
+            config.nodeClient.setOperator(TREASURY_ACCOUNT_ID, config.treasuryKey)
+            self.logger.info(`Sending 1 tinybar to account: ${accountId}`)
+            await this.accountManager.transferAmount(constants.TREASURY_ACCOUNT_ID, accountId, 1)
+          }
+        }
+      },
+      {
+        title: 'Finalize',
+        task: (ctx, _) => {
+          // reset flags so that keys are not regenerated later
+          self.configManager.setFlag(flags.generateGossipKeys, false)
+          self.configManager.setFlag(flags.generateTlsKeys, false)
+          self.configManager.persist()
+        }
+      }
+    ], {
+      concurrent: false,
+      rendererOptions: constants.LISTR_DEFAULT_RENDERER_OPTION
+    })
+
+    try {
+      await tasks.run()
+    } catch (e) {
+      self.logger.error(`Error in updating nodes: ${e.message}`, e)
+      this.logger.error(e.stack)
+      console.error(e)
+      throw new FullstackTestingError(`Error in updating nodes: ${e.message}`, e)
+    } finally {
+      await self.close()
+    }
+
+    return true
+  }
+
+  async delete (argv) {
+    const self = this
+
+    const tasks = new Listr([
+      {
+        title: 'Initialize',
+        task: async (ctx, task) => {
+          self.configManager.update(argv)
+
+          // disable the prompts that we don't want to prompt the user for
+          prompts.disablePrompts([
+            flags.app,
+            flags.chainId,
+            flags.localBuildPath,
+            flags.chartDirectory,
+            flags.devMode,
+            flags.endpointType,
+            flags.force,
+            flags.fstChartVersion,
+            flags.gossipEndpoints,
+            flags.grpcEndpoints
+          ])
+
+          await prompts.execute(task, self.configManager, NodeCommand.DELETE_FLAGS_LIST)
+
+          /**
+           * @typedef {Object} NodeDeleteConfigClass
+           * -- flags --
+           * @property {string} app
+           * @property {string} cacheDir
+           * @property {string} charDirectory
+           * @property {boolean} devMode
+           * @property {string} endpointType
+           * @property {string} fstChartVersion
+           * @property {string} gossipEndpoints
+           * @property {string} grpcEndpoints
+           * @property {string} keyFormat
```

Comment:
```suggestion
           * @property {string} keyFormat
```
I don't think we use gossipEndpoints or grpcEndpoints in node delete

Reply:
Removed